playbooks/operationalizing/branding_console.adoc

```diff
@@ -273,17 +273,12 @@ Use the css class in a template::
 <1> This value must be same as the css style in extension.css
```

Comment:
@Jooho this change looks unrelated to DNS. Did you intend to include it in this PR?